### PR TITLE
Add MoneroV (rx/v) algorithm [based on MoneroOcean/master]

### PR DESCRIPTION
**Pull request by Spudz76**
Latest code from MoneroOcean/xmrig master, only the MoneroV rx/v algorithm part, remastered only slightly (shifted some lines around for consistency)

Appears to also support opencl backend (no CUDA).

### CODE REVIEW

File: src/backend/opencl/cl/rx/randomx.cl [hunk 1]
```diff
@@ -8,6 +8,8 @@
 #include "randomx_constants_loki.h"
 #elif (ALGO == ALGO_RX_ARQMA)
 #include "randomx_constants_arqma.h"
+#elif (ALGO == ALGO_RX_V)
+#include "randomx_constants_v.h"
```
Comment (SChernykh) [inline src/backend/opencl/cl/rx/randomx.cl]: There's no need to add OpenCL constants for rx/v - they're exactly the same as rx/0, see https://github.com/xmrig/xmrig/pull/1434 for reference

Comment (SChernykh) [reply src/backend/opencl/cl/rx/randomx.cl]: Basically, you just need to change algo id in OclRxBaseRunner.cpp from RX_V to RX_0 and that will be enough for OpenCL support. Dataset is generated on CPU with correct Arong2d salt, everything else can just run with rx/0 code.

**Comment by xmrig** [comment]
@Spudz76 Can you explain what is MoneroV is, https://monerov.org/ and https://github.com/monerov/monerov seems already dead, but what is https://monerov.online/ and https://github.com/xmvdev/monerov

**Comment by SChernykh** [comment]
monerov.online is the new "rebooted" MoneroV, it trades on tradeogre instead of the old one now so it's kind of "official".

**Comment by xmrig** [comment]
Oh okay, found https://monerov.online/MoneroV-Swap.pdf

File: src/backend/opencl/cl/rx/randomx.cl (hunk 1 above)
Comment (xmrig) [reply src/backend/opencl/cl/rx/randomx.cl]: I made required changes.

**Comment by xmrig** [comment]
I was most short lived algorithm ever https://twitter.com/monerovofficial/status/1209509550501482496

**Pull request merged**